mm2src/coins/utxo/qrc20.rs

```diff
@@ -0,0 +1,690 @@
+use common::jsonrpc_client::{JsonRpcClient, JsonRpcRequest, RpcRes};
+use common::mm_number::MmNumber;
+use crate::eth::{ERC20_CONTRACT, u256_to_big_decimal, wei_from_big_decimal};
+use crate::SwapOps;
+use ethabi::Token;
+use ethereum_types::{H160, U256};
+use futures::{TryFutureExt, FutureExt};
+use gstuff::now_ms;
+use rpc::v1::types::H160 as H160Json;
+use std::str::FromStr;
+use super::*;
+
+const QRC20_GAS_LIMIT_DEFAULT: u64 = 250_000;
+const QRC20_GAS_PRICE_DEFAULT: u64 = 40;
+const QRC20_DUST: u64 = 0;
+
+#[derive(Debug, Deserialize, Eq, PartialEq)]
+pub struct TokenInfo {
+    name: String,
+    decimals: u8,
+    total_supply: u64,
+    symbol: String,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ExecutionResult {
+    pub output: BytesJson,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct ContractCallResult {
+    address: H160Json,
+    #[serde(rename = "executionResult")]
+    pub execution_result: ExecutionResult,
+}
+
+/// QTUM specific RPC ops
+pub trait QtumRpcOps {
+    fn blockchain_token_get_info(&self, token_addr: &H160Json) -> RpcRes<TokenInfo>;
+
+    fn blockchain_contract_call(&self, contract_addr: &H160Json, data: BytesJson) -> RpcRes<ContractCallResult>;
+}
+
```

Comment:
Cipi mentioned in issue that QTUM has recently added the docs for QRC20 integration: https://github.com/qtumproject/qtum-electrumx-server/blob/master/docs/qrc20-integration.md, please add the link here.

Reply:
Unfortunately there is not info about blockchain.contract.call.
I've added the link only for the blockchain.token.get_info